packages/db/src/schema.ts

```diff
@@ -33,6 +33,7 @@ export enum Bucket {
   phase0_proposerSlashing = 14, // ValidatorIndex -> ProposerSlashing
   phase0_attesterSlashing = 15, // Root -> AttesterSlashing
   capella_blsToExecutionChange = 16, // ValidatorIndex -> SignedBLSToExecutionChange
+  capella_blsToExecutionChangeCache = 17, // ValidatorIndex -> SignedBLSToExecutionChange
```

Comment:
You should be able to recycle the existing in-memory + DB pool. Why is this extra bucket required?

Reply:
umm i guess the current db pool before capella can be used as to recycle, i was going for caching any generic publishing errors but i guess its over kill

Reply:
the problem could be, once capella is hit, it would become unclear which ones are received from gossip and which ones have been submitted, as there is always a possibility of restarts while the gossip job is not complete, so would like to retain this as a separate repo

Reply:
we can clean it post capella

Reply:
It does not matter where an object is received. You persist to db changes that are still valid to be included in a block

Reply:
> the problem could be, once capella is hit, it would become unclear which ones are received from gossip and which ones have been submitted

That's a fair point. However, it's not a consensus issue to re-broadcast other validator messages. After thinking about it I'm not opposed to mark those objects in some way as "submitted pre-capella"